### PR TITLE
add Sumo Logic to data ouputs page

### PR DESCRIPTION
**Pull request by frankreno**
Adding Sumo Logic to the Data Outputs page for Log Management.  Please let me know if any more details are needed.

### CODE REVIEW

File: views/dataoutputs.erb [hunk 1]
```diff
@@ -22,7 +22,7 @@
              [
                ['ElasticSearch + Kibana', 'elasticsearch.png', 'http://docs.fluentd.org/articles/free-alternative-to-splunk-by-fluentd'],
                ['Splunk', 'splunk.jpg', 'http://docs.fluentd.org/v0.12/articles/out_splunk'],
-               [],
+               ['Sumo Logic', 'sumo-logic.png', 'https://github.com/SumoLogic/fluentd-output-sumologic'],
```
Comment (cosmo0920) [inline views/dataoutputs.erb]: You shouldn't add direct repository link.
We should create a document for SumoLogic in https://github.com/fluent/fluentd-docs.

Comment (frankreno) [reply views/dataoutputs.erb]: @cosmo0920 : will file a PR for that soon, thanks for the tip

Comment (cosmo0920) [reply views/dataoutputs.erb]: Ah, some of other links use direct repository link.
We can add it....

Comment (frankreno) [reply views/dataoutputs.erb]: Oh great, if we are good with that short term, I can happily get an official doc later...

**Comment by frankreno** [comment]
@cosmo0920 : can we get this merged?

**Pull request merged**